scripts/sanitizeme.pl

```diff
@@ -253,6 +253,34 @@ sub delete_sensitive_user_data {
   $dbh->do("DELETE FROM ts_funcmap");
   $dbh->do("DELETE FROM ts_job");
   $dbh->do("DELETE FROM ts_note");
+
+  # Reminders
+  $dbh->do('TRUNCATE TABLE reminders');
+
+  # Whine events
+  $dbh->do('DELETE FROM whine_events');
+
+  # Watched users by other users
+  $dbh->do('TRUNCATE TABLE watch');
```

Comment:
Any reason for `DELETE FROM` in some places but `TRUNCATE TABLE` in others?

Reply:
The DELETE FROM  is required when the table you are deleting values for have foreign key relationships to that table and the other tables have ON DELETE CASCADE setup. So when we do the DELETE FROM it automatically deletes the rows in the other tables. If nothing else links to the rows in the table, you can just do TRUNCATE TABLE. TRUNCATE TABLE is preferable when you can because it resets the auto increment counters back to zero.

Reply:
> The DELETE FROM is required when the table you are deleting values for have foreign key relationships to that table and the other tables have ON DELETE CASCADE setup. So when we do the DELETE FROM it automatically deletes the rows in the other tables. If nothing else links to the rows in the table, you can just do TRUNCATE TABLE. TRUNCATE TABLE is preferable when you can because it resets the auto increment counters back to zero.

Actually, it will error as well if you try to TRUNCATE TABLE and their are rows that are referenced to by other tables.